docs/snap-clone.md

```diff
@@ -3,14 +3,14 @@
 - [Prerequisite](#prerequisite)
 - [Create CephFS Snapshot and Clone Volume](#create-cephfs-snapshot-and-clone-volume)
   - [Create CephFS SnapshotClass](#create-cephfs-snapshotclass)
-  - [Create CephFS Snapshot](#create-cephfs-snapshot)
-  - [Restore CephFS Snapshot to a new PVC](#restore-cephfs-snapshot-to-a-new-pvc)
-  - [Clone CephFS PVC](#clone-cephfs-pvc)
+  - [Create CephFS Snapshot](#create-snapshot)
+  - [Restore CephFS Snapshot to a new PVC](#restore-snapshot-to-a-new-pvc)
+  - [Clone CephFS PVC](#clone-pvc)
 - [Create RBD Snapshot and Clone Volume](#create-rbd-snapshot-and-clone-volume)
   - [Create RBD SnapshotClass](#create-rbd-snapshotclass)
   - [Create RBD Snapshot](#create-rbd-snapshot)
-  - [Restore RBD Snapshot to a new PVC](#restore-rbd-snapshot-to-a-new-pvc)
-  - [Clone RBD PVC](#clone-rbd-pvc)
+  - [Restore RBD Snapshot to a new PVC](#restore-the-snapshot)
+  - [Clone RBD PVC](#clone-pvc)
```

Comment:
@humblec Looks like few links are not working.

Reply:
@Madhu-1 can you please list out ?

Reply:
* clone rbd PVC is link point to cephfs clone

Any specif reason to remove cephfs and names from the heading? an we keep both cephfs and rbd heading similar?

now this is how it looks
## cephfs
* create-cephfs-snapshot-and-clone-volume
* create-cephfs-snapshotclass
* create-snapshot
* restore-snapshot-to-a-new-pvc
* clone-pvc

## rbd
* create-rbd-snapshot-and-clone-volume
* create-rbd-snapshotclass
* create-rbd-snapshot
* restore-the-snapshot
* clone-pvc

Reply:
@Madhu-1 thanks... done.. ptal